src/parameterizations/vertical/MOM_diabatic_driver.F90

```diff
@@ -295,6 +297,28 @@ subroutine diabatic(u, v, h, tv, Hml, fluxes, visc, ADp, CDp, dt, Time_end, &
   integer :: i, j, k, m, is, ie, js, je, nz
   logical :: showCallTree ! If true, show the call tree
 
+  real, allocatable, dimension(:,:,:)    :: h_in  ! thickness before thermodynamics
+  real, allocatable, dimension(:,:,:)    :: t_in  ! temperature before thermodynamics
+  real, allocatable, dimension(:,:,:)    :: s_in  ! salinity before thermodynamics
+  real :: t_tend,s_tend,h_tend                  ! holder for tendencey needed for SPPT
+  real :: t_pert,s_pert,h_pert                  ! holder for perturbations needed for SPPT
+
+  if (G%ke == 1) return
+
+   ! save  copy of the date for SPPT if active
+  if (stoch_CS%do_sppt) then
```

Comment:
`if` block needs to indented

Reply:
> @jiandewang Since this PR will be merged to ufs-weather in a commit which changes baselines, I think it would be good to test the MOM6 PR separately with the current develop branch to confirm there are no changes due to the new stoch phys option.

@DeniseWorthen it doesn't change any existing baseline results, just adds a new test.

Reply:
> `if` statement needs to indented

The indentation is correct. Line 306 does not have a then.